type_data/migrations/5_data_consistency_fixes.rb

```diff
@@ -16,6 +16,13 @@
   insert(:name, "necrotic snake")
   insert(:name, "spectre")
   insert(:name, "wild dog")
+  insert(:name, %{brindle wild hound}) # Orcswold, Yegharren Plains
+  insert(:name, %{(?:black|mangy|spotted|mongrel) wild dog}) # Orcswold, Yegharren Plains
+  insert(:name, %{red centaur}) # Locksmehr Trail
+  insert(:name, %{solider ant}) # WL
+  insert(:name, %{coconut crab}) # KF
+  insert(:name, %{ogre sentry}) # KF
+  insert(:name, %{wharf rat}) # KF
```

Comment:
These should be inside of 33_osa_kf_npcs.rb with the other KF mobs that were added